internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -874,6 +879,19 @@ func (c *ClusterManager) buildObservatoriumDexSecretResource() *k8sCoreV1.Secret
 	}
 }
 
+func (c *ClusterManager) buildDataplaneServiceaccountSecretResource(cluster *api.Cluster) *k8sCoreV1.Secret {
```

Comment:
I find this name might be too generic:
* What is a ServiceAccount in the context of clusters_mgr?
* Is this "service account" used for multiple functionalities in the data plane cluster?

Reply:
It is the FSO service account, so yes it is used for multiple purposes. I'll try to find a better name for it.